sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -371,16 +371,20 @@ idents
     ;
 
 ident
-    : IDENTIFIER                                                                     #unquotedIdentifier
-    | quotedIdentifier                                                               #quotedIdentifierAlternative
-    | nonReserved                                                                    #unquotedIdentifier
-    | BACKQUOTED_IDENTIFIER                                                          #backQuotedIdentifier
-    | DIGIT_IDENTIFIER                                                               #digitIdentifier
-    | COLON_IDENT                                                                    #colonIdentifier
+    : unquotedIdent
+    | quotedIdent
     ;
 
-quotedIdentifier
```

Comment:
it was needed to remove quotes with `exitQuotedIdentifier`. we can do it in the `quotedIdentifier` visitor as well.